_posts/2017-12-25-css-in-javascript-with-styled-components.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: CSS-in-JavaScript with Styled Components 🎅🏻
+date: 2017-12-25
+---
+
+I started a new job recently and I am learning a looooooooot of new technologies, libraries, etc... One of them is [styled-components](https://www.styled-components.com/). I'll briefly talk about styled-components at the end of this article, what I want to highlight here is a new (to me) concept **CSS-in-JavaScript**.
+
+#### CSS-in-JavaScript
+
+A few months ago, I wrote an [article](https://caroleolivier.github.io/blog/2017/10/06/styling-react-app) about using [CSS modules](https://github.com/css-modules/css-modules). I really liked CSS modules because they allow me to locally scope CSS classes and easyly maintain stylesheets.
```

Comment:
Oops, I missed this: easily not ~easyly~

Reply:
oops. I installed a spell checker (should have done it ages ago really...)